.github/workflows/trivy.yml

```diff
@@ -0,0 +1,39 @@
+# Runs linter for Docker files
+name: Trivy
```

Comment:
This should be added to `pr_push.yml` as a requirement for the Build workflow, on the same level as Spellcheck.

Reply:
actually we can't do this right now. We did the same with CodeQL, due to issue we encountered with scorecard and permissions. I copy-pasted the comment from CodeQL workflow.